javascript/javascript1/week2/addCandy.js

```diff
@@ -0,0 +1,44 @@
+let boughtCandyPrices = [];
+let amountToSpend = Math.random() * 100;
+function addCandy(candyType, weight) {
+  if (candyType === "sweet") {
+    let price1 = 0.5 * weight;
+    boughtCandyPrices.push(price1);
```

Comment:
you can declare price before if, then just assign new value like this
```js
let price;
 if (candyType === "sweet") {
    price = 0.5 * weight;
    boughtCandyPrices.push(price);
```
or simply push the price directly
```js
if (candyType === "sweet") {
    boughtCandyPrices.push(0.5 * weight);
```